contenttype/utils.go

```diff
@@ -24,6 +24,15 @@ func IsJSONContentType(contentType string) bool {
 	return isContentType(contentType, JSONContentType)
 }
 
+// IsStringContentType determines if content type is string
+func IsStringContentType(contentType string) bool {
```

Comment:
This is OK for now. I wonder if we could eventually have a richer MimeType comparison logic based on hierarchy. Similar to this project: https://github.com/gabriel-vasile/mimetype

Reply:
I created an issue to track this discussion: https://github.com/dapr/components-contrib/issues/635